Added a validation for demisto.log in XSOAR linter (#2217)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Fixed an issue where an incorrect error was shown when the `id` of a content item differed from its `name` attribute.
* Fixed an issue where the `preserve_quotes` in ruamel_handler received an incorrect value @icholy

> * Added new validation to the `XSOAR-linter` in the **lint** command that demisto.log will not be used in code.

Reviewer: ```suggestion
* Added a new validation to the `XSOAR-linter` in the **lint** command for verifying that demisto.log is not used in code.
```
